install/ensure-correct-permissions-profiles-dir.sh

```diff
@@ -2,6 +2,12 @@
 
 # TODO: Remove this after the next hard-stop
 
+# Should only run when `$COMPOSE_PROFILES` is set to `feature-complete`
+if [[ "$COMPOSE_PROFILES" == "errors-only" ]]; then
+  echo "Skipping permissions check for profiles directory."
+  exit 0
```

Comment:
```suggestion
  exit 0
```
This prevents the next files sourced in `install.sh` to be executed, is this fine?

Reply:
Should I enclose this within the if clause instead of exiting?

Reply:
Yes, do not `exit 0` whatever you do :)